contracts/LosslessGovernance.sol

```diff
@@ -649,4 +649,24 @@ contract LosslessGovernance is ILssGovernance, Initializable, AccessControlUpgra
 
         emit LosslessClaim(reportTokens, _reportId, amountToClaim);
     }
+
+    /// @notice Claims the committee's tokens for a report if no committee member has voted
+    /// @dev This function should be called by Lossless Admin only after a report is resolved
+    /// @param _reportId The ID of the report for which the tokens are being claimed
+    function losslessAdminClaimCommitteeTokens(uint256 _reportId) public onlyLosslessAdmin {
```

Comment:
Let's not merge this to master but create another branch just for these changes.